### PR TITLE
Rebrand Phase One: Legacy Design Tokens

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
These changes update our soon-to-be-legacy design tokens to match the brand updates ARC is implementing in Phase One.

Note: Color tokens (e.g. 200) do not match those in Figma, since the new foundations anticipate a different schema.

I also have not updated any MUI variables to use new tokens yet; this is purely a palette swap while we determine what is acceptable for ARC/Odyssey 0.3.0.

### CODE REVIEW

**Comment by edburyenegren-okta** [comment]
@nickarcidy-okta thanks! I'll get updates in tomorrow. 

Broad note that these mappings are being applied to our old tokens, which introduces some review difficulty. The biggest is that the `###` naming application doesn't line up. For example, in the current token schema, `900` is expected where we're using `700` in the new palette. This is why the values are mismatched today.

I will look at the ARC variables this week and see if we can smooth some of that over. It'd be easier for everyone if our naming was on the same page.

**Comment by nickarcidy-okta** [comment]
Thanks Edbury, as long as the values match the mapping as outlined, that's my top prio. Naming conventions I'm totally flexible on as far as alignment goes. If you, Jose, and/or Taylor have opinions there, I defer to you!

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:17]: Figma has a Grey 300 as well, does that not exist in code currently?

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:27]: Should be: #6E6E6E

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:41]: 300 should be #9DAAF1

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:45]: 500 should be #4C64E1

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:49]: Blue 900 should be #22307C

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:63]: Green 500 should be #197F48

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:67]: Green 600 should be #0E562F

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:71]: Green 900 should be #0A4023

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:85]: Red 500 should be #D92300

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (taylorridgway-okta) [inline packages/odyssey-design-tokens/src/color/palette.json:89]: Red 900 should be #711200

**Review by taylorridgway-okta — CHANGES_REQUESTED** [review]

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (edburyenegren-okta) [reply packages/odyssey-design-tokens/src/color/palette.json:17]: Correct, there's no `300` in the current tokens.

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (KevinGhadyani-Okta) [inline packages/odyssey-design-tokens/src/color/palette.json:81]: Are the capital letters easier to copy outta Figma? :P

File: packages/odyssey-design-tokens/src/font.json
Comment (KevinGhadyani-Okta) [inline packages/odyssey-design-tokens/src/font.json:6]: Do we provide `Inter` in our library or is it something apps are expected to have?

File: packages/odyssey-storybook/src/guidelines/Colors.stories.mdx
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/guidelines/Colors.stories.mdx:8]: You can remove this now right? Or you mean deploying an update to this file?

**Review by KevinGhadyani-Okta — APPROVED** [review]
Just some questions. *NON-BLOCKIING*.

File: packages/odyssey-design-tokens/src/color/palette.json
Comment (edburyenegren-okta) [reply packages/odyssey-design-tokens/src/color/palette.json:81]: I'm going to downcase all of these before merge. 

Right now, it's useful for me to see which values are deviations and match them against Figma's output.

File: packages/odyssey-design-tokens/src/font.json
Comment (edburyenegren-okta) [reply packages/odyssey-design-tokens/src/font.json:6]: We don't provide `Inter`, so this stack will immediately fall back to system fonts if the downstream teams haven't provided it / it's not available locally.

File: packages/odyssey-storybook/src/guidelines/Colors.stories.mdx
Comment (edburyenegren-okta) [reply packages/odyssey-storybook/src/guidelines/Colors.stories.mdx:8]: I'll be removing this before merge. We need to update https://github.com/okta/odyssey/blob/develop/.github/workflows/ci.yml#L40 to include `odyssey-design-tokens` (possibly others).

**Review by ganeshsomasundaram-okta — APPROVED** [review]

**Review by taylorridgway-okta — APPROVED** [review]

**Pull request merged**
